core/Cargo.toml

```diff
@@ -48,8 +48,6 @@ serde = ["multihash/serde-codec", "dep:serde", "libp2p-identity/serde"]
 # More information: https://docs.rs/about/builds#cross-compiling
 [package.metadata.docs.rs]
 all-features = true
-rustdoc-args = ["--cfg", "docsrs"]
```

Comment:
Is there a way to verify that this isn't required anymore?

Reply:
It is mentioned in the documentation: https://docs.rs/about/builds#detecting-docsrs

It's also here in the code: https://github.com/rust-lang/docs.rs/blob/e579f2de083a650f6589c3133b5138f5d1915c7d/crates/metadata/lib.rs#L270-L276